packages/bpk-component-bottom-sheet/src/BpkBottomSheetInner-test.tsx

```diff
@@ -71,4 +71,24 @@ describe('BpkBottomSheetInner', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  it('should render correctly without title prop', () => {
+    const { asFragment, container } = render(
+      <BpkBottomSheetInner
+        id="my-bottom-sheet"
+        title=""
+        dialogRef={jest.fn()}
+        onClose={jest.fn()}
+        actionText="Dismiss"
+        onAction={jest.fn()}
+        exiting={false}
+      >
+        Bottom sheet content
+      </BpkBottomSheetInner>,
+    );
+
+    const titleElement = container.querySelector('h2');
+    expect(titleElement).toBeNull();
```

Comment:
I think it would be better to use `.not.toBeInTheDocument()`

i.e.
`expect(container.querySelector('h2')).not.toBeInTheDocument(); `